generator/src/test/java/com/scottlogic/deg/generator/generation/TestGenerationConfigSource.java

```diff
@@ -5,6 +5,7 @@
     public GenerationConfig.CombinationStrategyType combinationStrategy;
     public GenerationConfig.TreeWalkerType walkerType;
     public long maxRows = GenerationConfig.Constants.DEFAULT_MAX_ROWS;
+    public boolean profileValidation = GenerationConfig.Constants.DEFAULT_PROFILE_VALIDATION;
```

Comment:
See comment in `TreeTransformationIntegrationTests.java`

Reply:
I don't think the tests have anything to do with the profile validation so I think it should be disabled. Re: explicity, perhaps it will be better to create a method overload in the profile reader where it is set to disabled in just one place. Is that what you meant? Thanks.

Reply:
I think its fine with the method signature as it is; but be explicit about whether validation is on or not.

Reply:
changed.